### ISSUE TITLE
Speakers page accessibility

### ISSUE THREAD

**Issue by ttraenkler** [WebAssemblySummit/webassembly-summit.org#63]
As reported in #52 we need to add headline tags + alt image text to improve the accessibility of the speakers page - in particular add headline tags h1 (see also #61 for adding the missing title) and h2 and alt text for profile pictures.

**Comment by ttraenkler** [comment]
addressed in #67

**Comment by ttraenkler** [comment]
#71 integrated into this ticket

**Comment by ttraenkler** [comment]
fixed and deployed
